src/ln/channelmanager.rs

```diff
@@ -291,6 +301,8 @@ pub struct ChannelManager {
 }
 
 const CLTV_EXPIRY_DELTA: u16 = 6 * 24 * 2; //TODO?
+const CLTV_FAR_FAR_AWAY: u16 = 6 * 24 * 7; //TODO?
+const FINAL_NODE_TIMEOUT: u16 = 3; //TODO?
```

Comment:
I dont think this needs to be a new constant - channelmonitor::CLTV_CLAIM_BUFFER + HTLC_FAIL_TIMEOUT_BLOCKS should be right, I think.